src/content/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring.mdx

```diff
@@ -58,32 +65,119 @@ Grab the URL of a page you want to test and drop it in the **URL** field.
+
+<Steps>
+
+<Step>
+### Sign into your New Relic account [#steplogin]
+
+Once you've signed in to your New Relic account, go to **[one.newrelic.com > Synthetic monitoring > Create a monitor](https://one.newrelic.com/synthetics-nerdlets/monitor-create)**. Choose the **User step execution** tile. 
+
+</Step>
+
+<Step>
+### Create your step monitor [#step]
+
+<SideBySide>
+  <Side>
+    Give your monitor a name, choose how often your monitor runs its workflow, and select the location your monitor deploys from. 
+    
+    Go to **[one.newrelic.com > Synthetic monitoring > Create a monitor](https://one.newrelic.com/synthetics-nerdlets/monitor-create)**, then choose **User step execution**.
+
+  </Side>
+  <Side>
+<Video
+    type="wistia"
+    id="dum72g5b7z"
+/>
+    </Side>
+</SideBySide>
+
+</Step>
+
+<Step> 
+### Define your steps [#define]
+
+<SideBySide>
+  <Side>
+Mix and match from a selection of 12 prebuilt steps. Your monitor can be simple, like clicking buttons to hop through different pages, or you can test a journey that looks like:
+
+1. A customer goes to your web app 
+2. They type their email into a page modal
+3. They select from a dropdown and navigate to a different page
+4. They select something on this page, such as an item to purchase
+5. They fill out forms with secure credentials purchase that item, then submits those forms
+
+Whatever you choose to build, remember to click **Validate** before saving your monitor. Validating checks that the steps you've strung together run successfully.  
+  </Side>
+
+  <Side>
+<Video
+    type="wistia"
+    id="s4hd01vh9j"
+/>
+
+  </Side>
+</SideBySide>
+
+</Step> 
+
+<Step>
+
+### View your data [#stepdata] 
+
+<SideBySide>
+  <Side>
+When your monitor starts reporting, you can view your data in the synthetics **Summary page**. Go to **[one.newrelic.com > Synthetic monitoring](https://one.newrelic.com/synthetics-nerdlets)** > (Select a monitor) > (View your **Summary page**).
+
+  </Side>
+  <Side>
+<img
+    title="View your summary page"
+    alt="A screenshot of the summary page after a simple browser monitor reports data"
+    src={syntheticStepMonitorSummaryPage}
 />
 
-<figcaption>
-Go to **[one.newrelic.com > Synthetic monitoring](https://one.newrelic.com/synthetics-nerdlets)** (select a monitor) > (view your **Summary page**)
-</figcaption>  
 </Side>
+
 </SideBySide>
 
 </Step>
 
 </Steps>
 
-## Create a scripted browser monitor [#script]
+## Test site behavior with scripted monitoring [#script]
+
+Scripted browser monitors are a flexible monitor type that let you test website behavior. Scripting lets you deploy monitors that respond to your web apps like a customer might, choosing different paths across your site depending on conditions you've scripted. 
```

Comment:
"Scripted browser monitors are a flexible monitor type" reads a little weird to me. Maybe just cut out the "are a flexible monitor type" and just go straight to what SBMs actually do, more direct value prop.